oneflux_steps/ustar_cp_refactor_wip/launch.m

```diff
@@ -7,369 +7,162 @@
-% 	i=strmatch('NEE_or',int_L3,'exact'); NEE=data_L3(:,i); 
-% 	i=strmatch('Ta',int_L3,'exact'); Ta=data_L3(:,i); 
-% 	i=strmatch('PPFD',int_L3,'exact'); PPFD=data_L3(:,i); 
-% 	i=strmatch('Rg',int_L3,'exact'); Rg=data_L3(:,i); 
+            [Cp2,Stats2,Cp3,Stats3] = ... 
+                cpdBootstrapUStarTh4Season20100901 ...
+                    (t,NEE,uStar,T,fNight,fPlot,cSiteYr,nBoot); 
 
-    %by alessio ( by carlo)
-     % insert Dtime
-    nrPerDay = mod(numel(uStar),365);
-    if nrPerDay == 0; nrPerDay = mod(numel(uStar),364);end
-    t = 1 + (1 / nrPerDay);
-    for n2 = 2:numel(uStar); t(n2,1) = t(n2-1,1)+ (1 / nrPerDay);end
-    clear n2
-	
-	fNight=Rg<5; % flag nighttime periods
-	T=Ta;
-	
-%	Look at inputs.	
-	
-	fPlot=0; 
-	if fPlot;
-		plot(t,uStar,'.'); mydatetick(t,'Mo',4,1); pause;
-		plot(t,NEE,'.'); mydatetick(t,'Mo',4,1); pause;
-		plot(t,Ta,'.'); mydatetick(t,'Mo',4,1); pause;
-		plot(t,PPFD,'.'); mydatetick(t,'Mo',4,1); pause;
-		plot(t,Rg,'.'); mydatetick(t,'Mo',4,1); pause;
-	end;
-	
-%	Call uStarTh bootstrappng program (2 versions) 
-%	and assign annual Cp arrays.	
-	
-	fPlot=0; cSiteYr=strrep(d(n).name,'.txt','');%'CACa1-2001'; 
-	cSiteYr = strrep(cSiteYr, '_ut', '_barr');
-	nBoot=100;
-	
-    
-	% 4-season analysis
-    %by alessio
-	%if exist([cOut,'4-season_analysis_',strrep(cSiteYr,'.txt',''),'.mat']) == 0
-      
-        [Cp2,Stats2,Cp3,Stats3] = ... 
-            cpdBootstrapUStarTh4Season20100901 ...
-                (t,NEE,uStar,T,fNight,fPlot,cSiteYr,nBoot); 
+        % 	print -djpeg100 Plot1_4Season_CACa1-2001; 
 
-    % 	print -djpeg100 Plot1_4Season_CACa1-2001; 
 
-    [Cp,n,tW,CpW,cMode,cFailure,fSelect,sSine,FracSig,FracModeD,FracSelect] ... 
-        = cpdAssignUStarTh20100901(Stats2,fPlot,cSiteYr); 
-    if isempty(cFailure)
-        %save([cOut,'4-season_analysis_',strrep(cSiteYr,'.txt',''),'.mat'],'Cp2','Stats2','Cp3','Stats3',...
-        %    'Cp','n','tW','CpW','cMode','cFailure','fSelect','sSine','FracSig','FracModeD','FracSelect');
-        cSiteYr = strrep(cSiteYr, '.csv', '');
-        dlmwrite([output_folder,char(site),'_uscp_',char(year),'.txt'], Cp, 'precision', 8);
+            [Cp,n,tW,CpW,cMode,cFailure,fSelect,sSine,FracSig,FracModeD,FracSelect] ... 
+            = cpdAssignUStarTh20100901(Stats2,fPlot,cSiteYr); 
         
-        fid = fopen([output_folder,char(site),'_uscp_',char(year),'.txt'], 'a');
-        fprintf(fid, '\n;processed with ustar_mp 1.0 on %s\n', datestr(clock));
-        for i = length(notes):-1:1
-            fprintf(fid, ';%s\n', notes{i});
-        end
-        clear i
-        fclose(fid);
-        clear fid;
-        fprintf('ok\n');
-    else
-        error_str = [error_str; char(site),'_uscp_',char(year),' ',cFailure];
-        fprintf('%s\n', cFailure);
-        exitcode = 1;
-    end
-    % by alessio
-    %end    
-%     print -djpeg100 Plot2_4Season_CACa1-2001; 
-	
-% commented by alessio
-% 	% multi-season analysis

```

Comment:
Maybe we best preserve this comment block in launch.m for now so that it is kept documented somewhere for the future?

Reply:
Makes sense, I will add this back